src/screens/LanguageSelect/LanguageButton.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Button } from "react-native-elements";
+import PropTypes from "prop-types";
+import { styles } from "./styles";
+
+const LanguageButton = props => {
+  const { title, langKey, navigation } = props;
+  /* eslint-disable-next-line */
```

Comment:
What is a problem here with eslint?

Reply:
variable 'key' is not used anywhere, actual language selection will be added in the scope of task [UI integration](https://jira.epam.com/jira/browse/EPMCUXDA-6856)